### PR TITLE
Swagger Documentation of StandingInstructionHistoryApiResource.java

### PR DESCRIPTION
**Pull request by luckyman20**
GCI Task: 4870653989617664

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionHistoryApiResource.java [hunk 1]
```diff
@@ -67,14 +64,16 @@ public StandingInstructionHistoryApiResource(final PlatformSecurityContext conte
...
-    public String retrieveAll(@Context final UriInfo uriInfo, @QueryParam("sqlSearch") final String sqlSearch,
-            @QueryParam("externalId") final String externalId, @QueryParam("offset") final Integer offset,
-            @QueryParam("limit") final Integer limit, @QueryParam("orderBy") final String orderBy,
-            @QueryParam("sortOrder") final String sortOrder, @QueryParam("transferType") final Integer transferType,
-            @QueryParam("clientName") final String clientName, @QueryParam("clientId") final Long clientId,
-            @QueryParam("fromAccountId") final Long fromAccount, @QueryParam("fromAccountType") final Integer fromAccountType,
-            @QueryParam("locale") final String locale, @QueryParam("dateFormat") final String dateFormat,
-            @QueryParam("fromDate") final DateParam fromDateParam, @QueryParam("toDate") final DateParam toDateParam) {
+    @ApiOperation(value = "Standing Instructions Logged History", httpMethod = "GET", notes = "Example Requests :\n" + "\n" + "standinginstructionrunhistory\n" + "\n" + "standinginstructionrunhistory?orderBy=name&sortOrder=DESC\n" + "\n" + "standinginstructionrunhistory?offset=10&limit=50")
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionHistoryApiResource.java]: Add notes as same as description with the Example Request

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionHistoryApiResource.java [hunk 2]
```diff
@@ -67,14 +64,16 @@ public StandingInstructionHistoryApiResource(final PlatformSecurityContext conte
...
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", response = StandingInstructionHistoryApiResourceSwagger.GetStandingInstructionHistoryResponse.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionHistoryApiResource.java]: Refractor Inner class name to GetStandingInstructionRunHistoryResponse

Comment (Sanyam96) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionHistoryApiResource.java]: Swagger file is not giving the correct response on Swagger-UI

**Comment by luckyman20** [comment]
@Sanyam96 Updated!!

**Pull request merged**